### ISSUE TITLE
BulkCopyOptions with CheckConstraints = false not working with PostgreSQL DataProvider

### ISSUE THREAD

**Issue by YXMagnuMXY** [linq2db/linq2db#1140]
Hi guys!

I submitted an issue before about not working KeepIdentity with PostgreSQL, and you fixed it in last pre-release.
We need also the CheckConstraints to work, but it isn't working.

I am trying now to copy data from many tables in other database(let's call it secondDB).
I am starting a Transaction, and inserting the data from different tables to this database.
After that, i have an error that the constraint is violated at copy of the 7th table.
This happens because the transaction is not commited yet, and in table that contains this constraint reference, does not yet have the data needed.

I admit that "CheckConstraints" is handeling that, right?
Was "CheckConstraints" not implemented for PostgreSQL DataProvider?

**Comment by MaceWindu** [comment]
I suspect this option works only for SQL Server. We need to review and document all bulk copy options properly, but for now it is done for keep identity only.

Quick look at postgre sql  docs show that it should be possible to implement.

**Comment by MaceWindu** [comment]
As workaround for now I think you should disable constrains before BulkCopy call

**Comment by YXMagnuMXY** [comment]
Ok. Thank you. But you will fix this option in the next release, right?

**Comment by MaceWindu** [comment]
Yep, will review it and document/fix for all providers that could do it
